### PR TITLE
Update Scheduler name

### PR DESCRIPTION
**Pull request by Rajadeepan**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/741)

### CODE REVIEW

**Comment by Rajadeepan** [comment]
I signed it!

**Comment by k82cn** [comment]
/cc @gaocegege @richardsliu 

As we sync up, we'd like to work together on the batch scheduler part of Volcano :) 
We'll update related doc & code for more new features, e.g. Queue.

**Comment by gaocegege** [comment]
Some things that we need to know: Volcano scheduler is built on top of kube-batchd and maintained by @k82cn and his team. The Volcano scheduler will have some advanced features compared with kube-batchd, but share the same interface with kube-batchd. Thus I think we could use it.

/lgtm

**Comment by k82cn** [comment]
@gaocegege , thanks; yes, I'm the owner of kube-batch and volcano; we'll make sure both of them share the same API :)

File: content/docs/other-guides/job-scheduling.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ description = "How to schedule a job with gang-scheduling"
 weight = 30
 +++
 
-This guide describes how to use kube-batch to support gang-scheduling in 
+This guide describes how to use volcano scheduler to support gang-scheduling in
```
Comment (richardsliu) [inline content/docs/other-guides/job-scheduling.md]: Maybe add the volcano github link here instead of later, since readers may not be familiar with it.

Comment (k82cn) [reply content/docs/other-guides/job-scheduling.md]: +1

Comment (Rajadeepan) [reply content/docs/other-guides/job-scheduling.md]: Added the volcano scheduler link

**Pull request merged**